### PR TITLE
Fix #2: Update README.md

### LINKED ISSUE #2: Readme.md file
Create Readme.md file

Project name
Features being developed
Design links
Link to other repository
How to run the project?
Technology used
...anything else which is important.

### PR DESCRIPTION
**Pull request by aniketpatidar**
Resolves #2 

### Description
- Updated the README to include KlinicCon features, design, and setup instructions.

### Type of change
- [x] Documentation Update

### How to test this PR?
- Review the updated `README.md` file.

### CODE REVIEW

**Review by rt4914 — CHANGES_REQUESTED** [review]
@aniketpatidar have a look at this comment: https://github.com/rt4914/KlinicCon-Frontend/pull/15#issuecomment-2282734782

File: README.md [hunk 1]
```diff
@@ -1,8 +1,201 @@
...
+  </tbody>
+</table>
+
+## Contributing
+See [CONTRIBUTING.md](https://github.com/rt4914/KlinicCon-Frontend/blob/main/.github/CONTRIBUTING.md)
+
+## License
+This project is available as open source under the terms of the [MIT License](https://github.com/rt4914/KlinicCon-Frontend/blob/main/LICENSE.txt).
+
+## Contributors
```
Comment (rt4914) [inline README.md:175]: The contributor list should be in alphabetical order.

**Review by rt4914 — APPROVED** [review]
Really great work @aniketpatidar and also @KvngJamesNG for the earlier PR.

This is turning out to be a great implementation.

File: README.md (hunk 1 above)
Comment (rt4914) [reply README.md:175]: @aniketpatidar Once you have made this change, you can resolve this specific comment which will allow you to merge this PR.

Comment (aniketpatidar) [reply README.md:175]: I've updated the contributor list to be in alphabetical order.

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

**Pull request merged**